### ISSUE TITLE
Hello World example for medium-editor

### ISSUE THREAD

**Issue by scripting** [yabwe/medium-editor#743]
When I was beginning work on medium-editor, I wished there was a simple Hello World app, that showed the basics of how to get an editor going. 

Before I move on to other projects I wanted to put a little time into creating such an app. It might help other people in the future.

https://gist.github.com/scripting/fef457e14000b45e75f8

Also if people more experienced with medium-editor see ways to improve this example, please let me know. It would help me improve MyWord Editor. ;-)

You can see this example app running here:

http://myword.io/testing/hellomediumeditor.html

**Comment by j0k3r** [comment]
We usually use the demo page for that: https://yabwe.github.io/medium-editor/
It shows a simple editor using all default options. It's a good starting point for your test.

Also, if you clone the repo, there are more complex examples in the [demo folder](https://github.com/yabwe/medium-editor/tree/master/demo) which are really great to understand some options.

Your *hello-world* uses jQuery, we try to be dependencies free (at least from jQuery).

But thanks for the example by the way, localStorage is a good idea to keep the mediumEditor content :wink:

**Comment by scripting** [comment]
Thanks for the quick and excellent feedback.

First, I could easily remove the jquery code, if you all think it's
important.

The problem with the previous example code is that I didn't see clearly how
to assign the text to to editor initially. I didn't see anything in the api
that did that, and it isn't something that the canonical example does.

 I almost didn't use this editor because of that stumbling block.

Now that I understand how it works it seems odd, even to me, that I had the
problem. But it's a fact, I did. Others might too. A hello world app
should, imho, model the app the dev is likely to want to create, as simply
as possible.

Again thanks, and I'm glad I was in the end able to use medium-editor.



On Monday, July 27, 2015, Jeremy Benoist <notifications@github.com> wrote:

> Closed #743.
>
> —
> Reply to this email directly or view it on GitHub.
>


-- 
Typed on an iPad with fat fingers.

**Comment by j0k3r** [comment]
I understand there were no easy way to override the content of the editor programmatically.
But since a week there is now a `setContent` available (see #736)!

**Comment by scripting** [comment]
Very interesting! :-)

The reason this is such an important block is that it comes as you're getting started. When the whole thing is the most confusing, and when figuring out where to start is your first and really only problem.

I find that many open source projects (including some of my own) fall down on this, failing to empathize with the first steps of a developer who is unfamiliar with the project. That's why I thought it was important to put this back into the mix before I moved on. I will never be a total medium-editor newbie again. 

I have another bit of feedback, btw -- there are lots of tools out there that share this name. Makes it hard to search for questions about it. 

This project has more traction relative to the others. But it wasn't among the first set that I found when I was looking earlier this year, yet it did exist at the time. 

Not sure exactly what I'd recommend doing about it. Maybe come up with a standard #hashtag to refer to questions related to this project, something more specific than "#mediumeditor".
